Update faculty affiliations

eelkef opened the pull request:
Updated faculty hires, University of Nevada, Reno

Commit:
Update faculty affiliations

Updated faculty hires, University of Nevada, Reno

emeryberger commented on faculty-affiliations.csv:
```diff
@@ -8239,25 +8239,30 @@ ThanhVu Nguyen , University of Nebraska
 Shamik Sengupta , University of Nevada
+Siming Liu , University of Nevada
 Sushil J. Louis , University of Nevada
+Tin C Nguyen, University of Nevada
```
This name doesn't appear in DBLP.

emeryberger commented on faculty-affiliations.csv:
```diff
@@ -8239,25 +8239,30 @@ ThanhVu Nguyen , University of Nebraska
+Siming Liu , University of Nevada
 Sushil J. Louis , University of Nevada
+Tin C Nguyen, University of Nevada
+Yaakov Varol, University of Nevada
```
This name is also not correct (appears to be missing an L.)

emeryberger commented on faculty-affiliations.csv:
```diff
@@ -8239,25 +8239,30 @@ ThanhVu Nguyen , University of Nebraska
 Frederick C. Harris Jr. , University of Nevada
 George Bebis , University of Nevada
 Hung Manh La , University of Nevada
+Hui Lin , University of Nevada
```
Please update name to point to actual person on DBLP by adding the appropriate suffix (e.g., 0001).

See http://dblp.uni-trier.de/pers/hd/l/Lin:Hui

emeryberger commented:
Please check to make sure all of these appear in DBLP (or modify them so they do). Also, I'd appreciate it if you would add the home pages to `homepages.csv` in this PR.

Commit:
update faculty affiliations

fixed  author names. Removed Hui Lin for now until we can create a separate profile for him on the dblp.
